news/2022-10-10-halloween-fanart-contest.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "Halloween 2022 Fanart Contest"
+date: 2022-10-11 16:00:00 +0000
+---
+
+Spooky month has arrived. This means one thing: time for the artists of the osu! community to demonstrate their frightening skill!
+
+![](https://assets.ppy.sh/contests/154/header.jpg)
+
+Yep, it's that time of year again. The time compression that started back in 2020 never really stopped and we're now suddenly just in October again with a burning question waiting to be answered: **do you like to draw?**
+
+If your answer to that is *yes*, then take up your implement of choice and **draw any of the osu! mascots (or community mascots) in a Halloween theme or setting** for fun and glory!
+
+You've got **10 days** to pluck some magnificently mortifying artwork out of your noggin and make it real in the medium of your choice while also following the rules listed out below. **Make sure you read them!** We've added one regarding the use of AI art generation tools since we know they're all the (fiery)rage at the moment.
```

Comment:
i'm sorry but i'm gonna require an extensive explanation for why fieryrage is relevant here

Reply:
he's not, he snuck in to this because I mistakenly think I am funnier than I actually am